app/views/va_facilities/show.html.erb

```diff
@@ -0,0 +1,61 @@
+<% provide :main_classes, 'bg-gray-2' %>
+<% provide :footer_classes, 'bg-gray-2' %>
+
+<% provide :head_tags do %>
+  <%= javascript_include_tag 'facility_directory', 'data-turbolinks-track': 'reload' %>
+<% end %>
+<%= render partial: 'va_facilities/facility_map'%>
+<section class="usa-section padding-y-0 grid-col-12">
+  <div class="grid-container position-relative grid-col-12">
+    <div class="grid-row grid-gap">
+      <div class="grid-col-12">
+        <h1 class="font-sans-2x1 desktop:margin-top-5 desktop:margin-bottom-3">
+          <%= @va_facility.official_station_name %>
+        </h1>
+        <div class="grid-col-12">
+          <div class="grid-col-6" style="float: left;"> This facility has created <%= @practice_results_count %> practices and has adopted <%= @adoptions_count %> practices.  It's a<% if @va_facility.rurality == "U" || @va_facility.rurality == "I" %>n<% end %>
+            <% if @va_facility.rurality == "R" %>
+              rural
+            <% end %>
+            <% if @va_facility.rurality == "U" %>
+              urban
+            <% end %>
+            <% if @va_facility.rurality == "H" %>
+              highly rural
+            <% end %>
+            <% if @va_facility.rurality == "I" %>
+              isolated
+            <% end %>
+            <%= @va_facility.fy17_parent_station_complexity_level %>
+            facility and serves veterans in <%= @va_facility.mailing_address_city %>.  <% unless @va_facility.station_number_suffix_reservation_effective_date.blank? %>This facility opened in <%= @va_facility.station_number_suffix_reservation_effective_date.to_s[0..3] %>.<% end %>
+            <div class="desktop:margin-top-2 text-bold">
+              Address
+            </div>
+            <div class="desktop:margin-top-1">
+              <%= @va_facility.official_station_name %>
+            </div>
+            <div class="desktop:margin-top-1">
+              <%= @va_facility.street_address %>
+            </div>
+            <div class="desktop:margin-top-1">
+              <%= @va_facility.street_address_city %>, <%= @va_facility.street_address_state %> <%= @va_facility.street_address_zip_code %>
+            </div>
+            <div class="desktop:margin-top-2">
```

Comment:
I don't think we need all of these `desktop:` calls. You can just do `margin-top-1`.